crates/extensions/c8y_mapper_ext/src/service_monitor.rs

```diff
@@ -68,55 +68,55 @@ mod tests {
         r#"{"pid":"1234","status":"up"}"#,
         "c8y/s/us/test_device:device:main:service:tedge-mapper-c8y",
         r#"104,up"#;
-        "service-monitoring-thin-edge-device"
+        "service monitoring thin-edge device"
```

Comment:
Not related, I just failed to spot (reading the code) that this was a test name and not just more data when it was kebab-case.